layers/census.py

```diff
@@ -1,3 +1,4 @@
+import csv
```

Comment:
Is this in error? it looks unused

Reply:
No, I happened to notice the import was incorrectly in the main file and missing here. We do `csv.DictReader` below